examples/python/README.md

```diff
@@ -3,14 +3,42 @@ The simplest example is [`minimal`](minimal/minimal.py). You may want to start t
 
 Read more about our examples at <https://www.rerun.io/examples>.
 
-## Setup
+
+## Running our examples with `pixi`
+The easiest way of running our examples from within the Rerun repository is to use [pixi](https://github.com/prefix-dev/pixi?tab=readme-ov-file#installation).
+
+### Building from source
+
+```sh
+pixi shell
+pixi run py-build
+pip install examples/python/minimal
+minimal
```

Comment:
well that's not really the intended way. All examples are already installed in the `examples` environment, which just needs to have a compatible rerun-sdk there, so:

```
pixi run -e examples py-build
pixi run -e examples minimal
```